botnim/cli.py

```diff
@@ -152,6 +154,107 @@ def assistant(assistant_id, openapi_spec, rtl, environment):
 # Add evaluate command to main CLI
 cli.add_command(evaluate)
 
+@cli.command(name='process-document')
+@click.argument('input_html_file')
+@click.argument('output_base_dir')
+@click.option('--content-type', default='סעיף')
+@click.option('--environment', default='staging')
+@click.option('--model', default='gpt-4.1')
+@click.option('--max-tokens', type=int, default=None)
+@click.option('--dry-run', is_flag=True)
+@click.option('--overwrite', is_flag=True)
+@click.option('--generate-markdown', is_flag=True)
+@click.option('--mediawiki-mode', is_flag=True)
+def process_document_cmd(input_html_file, output_base_dir, content_type, environment, model, max_tokens, dry_run, overwrite, generate_markdown, mediawiki_mode):
+    """Run the full document processing pipeline."""
+    argv = [
+        'process_document.py',
+        input_html_file,
+        output_base_dir,
+        '--content-type', content_type,
+        '--environment', environment,
+        '--model', model,
+    ]
+    if max_tokens:
+        argv += ['--max-tokens', str(max_tokens)]
+    if dry_run:
+        argv.append('--dry-run')
+    if overwrite:
+        argv.append('--overwrite')
+    if generate_markdown:
+        argv.append('--generate-markdown')
+    if mediawiki_mode:
+        argv.append('--mediawiki-mode')
+    sys.argv = argv
+    process_document.main()
+
+@cli.command(name='extract-structure')
+@click.argument('input_file')
+@click.argument('output_file')
+@click.option('--environment', default='staging')
+@click.option('--model', default='gpt-4.1')
+@click.option('--max-tokens', type=int, default=None)
+@click.option('--pretty', is_flag=True)
+@click.option('--mark-type', default=None)
+def extract_structure_cmd(input_file, output_file, environment, model, max_tokens, pretty, mark_type):
+    """Extract hierarchical structure from HTML using OpenAI API."""
+    argv = [
+        'extract_structure.py',
+        input_file,
+        output_file,
+        '--environment', environment,
+        '--model', model,
+    ]
+    if max_tokens:
+        argv += ['--max-tokens', str(max_tokens)]
+    if pretty:
+        argv.append('--pretty')
+    if mark_type:
+        argv += ['--mark-type', mark_type]
+    sys.argv = argv
+    extract_structure.main()
```

Comment:
why not call the function instead of this weird hack?

(same for the two below)